lib/spack/docs/packaging_guide.rst

```diff
@@ -4284,10 +4296,18 @@ stage **after** the software is installed to the package's metadata
 directory. The result is the following directory and files will be
 available for use in stand-alone tests:
 
-* ``join_path(self.install_test_root, 'tests')`` along with its files and subdirectories
+* ``join_path(self.install_test_root, 'tests')`` along with its files
+  and subdirectories
 * ``join_path(self.install_test_root, 'examples', 'foo.c')``
 * ``join_path(self.install_test_root, 'examples', 'bar.c')``
 
+.. note::
+
+   You **could** build and run the examples in place, which means
```

Comment:
Maybe -- assuming the user has write access to the file system. Seems to me that you should strongly recommend copying to the stage directory.

Reply:
OR we could automatically copy the cached files to a subdirectory of the spec's stage directory like we already do with the custom files (https://spack.readthedocs.io/en/latest/packaging_guide.html#adding-custom-files).

What do you think @becker33 ?

Reply:
FYI. Going to remove this part of the change since it has been re-worked in #23713